lib/pages/add_task.dart

```diff
@@ -33,28 +29,30 @@ class _AddTaskScreenState extends State<AddTaskScreen> {
           Padding(
             padding: const EdgeInsets.fromLTRB(12, 12, 12, 12),
             child: TextField(
-              key: const Key('Add a task text field'),
-              controller: _addTask,
+              key: const Key('add_a_task_text_field'),
+              style: TextStyle(color: iconColor),
+              controller: addTask,
               decoration: InputDecoration(
                 labelText: 'e.g. walk the dog, do the dishes...',
                 labelStyle: TextStyle(color: white, fontSize: 14),
               ),
             ),
           ),
           IconButton(
-              key: const Key('Add Task Button'),
-              icon: Icon(
-                Icons.add,
-                color: white,
-              ),
-              onPressed: () {
-                context
-                    .read<TaskCubit>()
-                    .addTask(task: Task(name: _addTask.text));
-                _addTask.clear();
-                context.go('/');
-              },
-              style: ElevatedButton.styleFrom(backgroundColor: black)),
+            key: const Key('add_task_button'),
+            icon: Icon(
+              Icons.add,
+              color: white,
+            ),
+            onPressed: () {
+              context.read<TaskCubit>().addTask(
+                    task: Task(name: addTask.text, status: TaskStatus.todo),
+                  );
+              addTask.clear();
+              context.go('/');
+            },
```

Comment:
Consider factoring this out to a separate, private function.

Reply:
done